.circleci/config.yml

```diff
@@ -24,10 +24,13 @@ jobs:
           key: v1-node-tslib-deps-{{ checksum "tslib/yarn.lock" }}
       - run: cd admin; yarn install --frozen-lockfile
       - run:
-          name: Install JUnit coverage reporter
+          name: Admin lint and ts compile
```

Comment:
I don't see any compilation steps in `yarn lint`

Reply:
`yarn lint` now does `standard && tsc`, see the `admin/package.json` edits

Reply:
I saw this later on in the PR... so now the name should be lintAndBuild?

Reply:
i left as-is as we are really only using `tsc` for code quality right now.  However, i think 2 distinct items would be clearer, such as `lint` and `lintAndBuild`.
let me know if you think otherwise.